packages/authentication/azure/pyproject.toml

```diff
@@ -21,8 +21,7 @@ classifiers = [
 homepage = "https://github.com/microsoft/kiota#readme"
 repository = "https://github.com/microsoft/kiota-python"
 documentation = "https://microsoft.github.io/kiota/"
-packages = [{include = "kiota_authentication_azure"}]	
-include = ["CHANGELOG.md", "LICENSE"]
+packages = [{include = "kiota_authentication_azure"}]
```

Comment:
there was a trailing whitespace `\t` that I removed, if you're wondering what this was 🙂